hadoop-client-modules/hadoop-client-integration-tests/src/test/java/org/apache/hadoop/example/ITUseMiniCluster.java

```diff
@@ -44,6 +44,10 @@
 import org.apache.hadoop.hdfs.web.WebHdfsTestUtil;
 import org.apache.hadoop.hdfs.web.WebHdfsConstants;
 
+import org.apache.hadoop.yarn.server.MiniYARNCluster;
```

Comment:
should go straight after line 45 - no gap

Reply:
will fix. I was trying to follow the existing style though which separates imports based on package.